src/StartPage.elm

```diff
@@ -0,0 +1,72 @@
+module StartPage exposing (..)
```

Comment:
Don't expose everything.
Just `Msg`, `Model`, `init`, `update`, `view` should be enough.